assets/kube-mixins/additional-prometheus-rule.yaml

```diff
@@ -0,0 +1,330 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    app.kubernetes.io/managed-by: cluster-monitoring-operator
+    app.kubernetes.io/part-of: openshift-monitoring
+    prometheus: k8s
+    role: alert-rules
+  name: additional-prometheus-rules
```

Comment:
Ditto, can we find a clearer name? Or maybe split unrelated rule groups into more PrometheusRule resources?

Reply:
> Ditto, can we find a clearer name? Or maybe split unrelated rule groups into more PrometheusRule resources?

Part of it was moved to `cluster-monitoring-operator` PrometheusRule and part into control plane objects.

> I think it would be more clear to move the test to pkg/tasks/clustermonitoringoperator.go

I agree. It was moved into `pkg/tasks/controlplane.go`